hackforla/VRMS
# Add mvp docker container

The mvp container was not working before, and this adds the mvp container. #390 

How to test:
1. Before pulling changes, run `docker-compose build` and then `docker-compose up`. Note that you cannot access the mvp instance. 
2. Pull the changes. 
3. Run the same commands and navigate to http://localhost:3200 and note that the mvp is present.
